src/tools/verificarToken.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
const express = require("express")
```